WordPress/gutenberg
# Sort order: Use unicode characters instead of svg icons

## What?
In data views table layout; use unicode characters instead of svg icons to communicate sort order. 

## Why?
Feels more integrated and lighter weight. 

## Testing Instructions
1. Enable data views experiment
2. Go to manage pages
3. Adjust the sort order by clicking a column header
4. Notice the unicode character replaces the svg icon

| Before | After |
| --- | --- |
|  <img width="229" alt="Screenshot 2023-12-06 at 16 33 24" src="https://github.com/WordPress/gutenberg/assets/846565/ff590dfa-f624-4f02-8ddc-b34fd4230e9c"> | <img width="301" alt="Screenshot 2023-12-06 at 16 32 44" src="https://github.com/WordPress/gutenberg/assets/846565/39cb9757-78e3-4398-bff8-cec9b17d7347"> |
